src/components/calendar/google_api.tsx

```diff
@@ -9,9 +9,14 @@ const googleCalendar = google.calendar({ version: "v3", auth: apiKey });
 // Refresh events every 5 minutes minus 10 seconds to never force a user to wait.
 var lastUpdated = new Date();
 const rateLimitMinutes = 5;
+const refreshRate = 60000 * rateLimitMinutes - 10000;
+if (refreshRate <= 0) {
+  throw new Error("Rate limit is zero or negative.");
+}
+
```

Comment:
we do not speak about this skill issue

Reply:
it shall go down in history, never to be repeated...